legacy-mongo-shell/Dockerfile

```diff
@@ -25,9 +25,15 @@ RUN python3 buildscripts/scons.py install-mongo
 FROM mongo:6.0.5
 
 COPY --from=build /mongo/build/install/bin/mongo /usr/bin
-RUN ln -sv /usr/bin/mongo /mongo
 
-WORKDIR /
+RUN mkdir -p /tests/mongo
+
+RUN ln -sv /usr/bin/mongo /tests/mongo/mongo
+
+# We need to set WORKDIR as the root directory of the repository so that various
```

Comment:
Ok, but there is no repository. In that second stage we copy only `mongo` binary. Git checkout does not exist in that stage

Reply:
Yes I see, it relies on mounting the `jstest` directory in our docker-compose.yml file. This was needed because they re-organized this directory it was pulling the older one.

Reply:
Maybe we can just checkout a newer branch? I can try that.

Reply:
Sorry, I don't understand